diff --git a/init/init.c b/init/init.c
index acc47dd..cd9d186 100644
--- a/init/init.c
+++ b/init/init.c
@@ -199,7 +199,7 @@ struct init_action {
 
 static struct init_action *init_action_list = NULL;
 
-static const char *log_console = VC_5;
+static const char *log_console = NULL;
 
 enum {
 	L_LOG = 0x1,
@@ -522,9 +522,6 @@ static pid_t run(const struct init_action *a)
 		}
 	}
 
-	/* Log the process name and args */
-	message(L_LOG, "starting pid %d, tty '%s': '%s'",
-			getpid(), a->terminal, a->command);
 
 	/* Now run it.  The new program will take over this PID,
 	 * so nothing further in init.c should be run. */
