diff --git a/Makefile b/Makefile
index 00c6085..f4b2e8c 100644
--- a/Makefile
+++ b/Makefile
@@ -154,7 +154,7 @@ CFLAGS ?= ${COMMON_FLAGS} \
 else
 # FIRMWARE_ARCH not defined; assuming local compile.
 CC ?= gcc
-CFLAGS += -DCHROMEOS_ENVIRONMENT -Wall -Werror ${DEBUG_FLAGS}
+CFLAGS += -DCHROMEOS_ENVIRONMENT -Wall ${DEBUG_FLAGS}
 endif
 
 ifneq (${DEBUG},)
@@ -418,12 +418,8 @@ UTILLIB_SRCS = \
 	host/lib/file_keys.c \
 	host/lib/fmap.c \
 	host/lib/host_common.c \
-	host/lib/host_key.c \
 	host/lib/host_keyblock.c \
 	host/lib/host_misc.c \
-	host/lib/util_misc.c \
-	host/lib/host_signature.c \
-	host/lib/signature_digest.c
 
 UTILLIB_OBJS = ${UTILLIB_SRCS:%.c=${BUILD}/%.o}
 ALL_OBJS += ${UTILLIB_OBJS}
@@ -523,7 +519,6 @@ CGPT_OBJS = ${CGPT_SRCS:%.c=${BUILD}/%.o}
 
 ALL_OBJS += ${CGPT_OBJS}
 
-CGPT_WRAPPER = ${BUILD}/cgpt/cgpt_wrapper
 
 CGPT_WRAPPER_SRCS = \
 	cgpt/cgpt_nor.c \
diff --git a/firmware/stub/vboot_api_stub_sf.c b/firmware/stub/vboot_api_stub_sf.c
index 3c5ec85..0b75606 100644
--- a/firmware/stub/vboot_api_stub_sf.c
+++ b/firmware/stub/vboot_api_stub_sf.c
@@ -5,7 +5,8 @@
  * Stub implementations of firmware-provided API functions.
  */
 
-#include <execinfo.h>
+#define backtrace(a,b) (0)
+#define backtrace_symbols_fd(a,b,c)
 #include <stdint.h>
 
 #define _STUB_IMPLEMENTATION_
